### ISSUE TITLE
Module @braid/vue-formulate/nuxt not found ?

### ISSUE THREAD

**Issue by golubvladimir** [wearebraid/vue-formulate#364]
I installed by npm : npm install @braid/vue-formulate
Add to nuxt config:
```
  buildModules: [
    '@braid/vue-formulate/nuxt'
  ],
```
also try:
```
  buildModules: [
    '@braid/vue-formulate/dist/nuxt'
  ],
```

I have error: 
Cannot find module '@braid/vue-formulate/nuxt'
or
Cannot find module '@braid/vue-formulate/dist/nuxt' 

Nuxt version: 
"nuxt": "^2.14.6",

**Comment by andrew-boyd** [comment]
@golubvladimir Whoops! Right you are. Those docs got merged but the PR is still open against the main Vue Formulate codebase. For now, I'm reverting the docs. Look for a 2.5.1 release soon!

**Comment by dosstx** [comment]
Was there a solution for this or do we wait until 2.5.1 release?

**Comment by justin-schroeder** [comment]
The docs for the module were published pre maturely (and have been reverted). You can still install with Nuxt just fine using a plugin. If you follow the current docs you should be good to go.

**Comment by dosstx** [comment]
Just wanted to report that this now works for me. Thanks for the update, Justin!

**Comment by justin-schroeder** [comment]
This is now included in the `release/2.5.1` branch which should be published today
